UserGuide.md

```diff
@@ -385,8 +385,6 @@ Gson has built-in serializers and deserializers for commonly used classes whose
 
 For many more, see the internal class [`TypeAdapters`](gson/src/main/java/com/google/gson/internal/bind/TypeAdapters.java).
 
-You can also find source code for some commonly used classes such as JodaTime at [this page](https://sites.google.com/site/gson/gson-type-adapters-for-common-classes-1).
```

Comment:
Simply removed this because Joda-Time is probably not that widely used anymore since the same classes exist since Java 8. And the linked page apparently only included code for two classes, see https://web.archive.org/web/20221102153855/https://sites.google.com/site/gson/gson-type-adapters-for-common-classes-1.